.travis.yml

```diff
@@ -7,6 +7,8 @@ branches:
   - master
 os:
 - linux
+before_install:
+- yarn global install -g grunt-cli
```

Comment:
Could you put this in dev dependencies instead ?

Reply:
done